dev/com.ibm.ws.jaxrs.2.0.common/src/org/apache/cxf/jaxrs/utils/JAXRSUtils.java

```diff
@@ -1435,9 +1437,11 @@ public Object run() throws IOException, WebApplicationException {
                 });
             } catch (PrivilegedActionException e) {
                 Exception e1 = e.getException();
-                if (e1 instanceof IOException)
+                if (e1 instanceof JsonParseException) {
+                    throw new BadRequestException(e1);
+                } else if (e1 instanceof IOException) {
                     throw (IOException) e1;
-                else
+                } else
```

Comment:
You can just remove the `else` keyword here.

Reply:
I prefer else blocks for uniformity (and brackets)